test/integration/smoke/test_kubernetes_clusters.py

```diff
@@ -568,24 +550,6 @@ def test_08_upgrade_kubernetes_ha_cluster(self):
         self.debug("Kubernetes cluster with ID: %s successfully upgraded" % k8s_cluster.id)
         return
 
-    @attr(tags=["advanced", "smoke"], required_hardware="true")
-    @skipTestIf("hypervisorNotSupported")
-    def test_09_delete_kubernetes_ha_cluster(self):
-        """Test to delete a HA Kubernetes cluster
-
-        # Validate the following:
-        # 1. deleteKubernetesCluster should delete an existing HA Kubernetes cluster
-        """
-        if self.setup_failed == True:
-            self.fail("Setup incomplete")
-        if self.default_network:
-            self.skipTest("HA cluster on shared network requires external ip address, skipping it")
-        global k8s_cluster
-        k8s_cluster = self.getValidKubernetesCluster(1, 3)
-
-        self.debug("Deleting Kubernetes cluster with ID: %s" % k8s_cluster.id)
-        return
-
```

Comment:
@weizhouapache these tests removed are not relevant now?

Reply:
@sureshanaparti 
the upgrade test contains the following steps
- create
- upgrade
- delete

We do not need to run create/delete separately.

Reply:
> @sureshanaparti the upgrade test contains the following steps
> 
> * create
> * upgrade
> * delete
> 
> We do not need to run create/delete separately.

ok @weizhouapache, got it